hvac/api/auth_methods/jwt.py

```diff
@@ -39,6 +39,7 @@ def configure(
         jwks_ca_pem=None,
         jwt_validation_pubkeys=None,
         bound_issuer=None,
+        namespace_in_state=None,
```

Comment:
Please move this parameter to the bottom of the list, that way clients supplying values positionally will not experience a breaking change.

Reply:
Makes sense. I moved it before `path`, let me know if that's fine.

Reply:
Unfortunately we need to move it _after_ `path` since `path` can still be supplied positionally.

I would like to go through the entire library and make most of the parameters keyword-only, so that we can avoid this in the future (it comes up in almost every PR on existing methods), but I just haven't found the time to write up the policy and rollout plan (it's a breaking change, so I want to do it over the course of a few major releases, with plenty of warning for anyone using the arguments positionally).

Reply:
Yeah I wasn't sure. I always use `*kw` anyways, it's safer.